packages/bitget-wallet/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@near-wallet-selector/bitget-wallet",
+  "version": "8.5.3",
```

Comment:
Can you update the version here to match with the latest version of wallet-selector (v8.8.0)?